src/spec/lib/ConfigSpec.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Config} from '../../lib/Config';
+
+describe('config', () => {
+  describe('#constructor', () => {
```

Comment:
So you could use mock-fs to test out loading from file and such (I don't know much about it, just a cursory Google search).

Reply:
I'll merge this and create another PR introducing a mock library. We need it for more thorough tests in other places too.